variablecompany/bite
# feat: create the QR screen and NFC screen

## Describe your changes
This PR will add a new logic for scanning the QR code and the NFC scan, this will connect in the future with the logic of the menu appearing at the bottom of the screen.

## How to test
- Test on Android and iOS
- Go on to press the scan button
- If your device supports the NFC you will see the animation for the NFC
- if your device doesn't support NFC technology you will see the QRCode

## Checklist before requesting a review
- [x] Will this be part of a product update?
